comparisontool/views.py

```diff
@@ -40,19 +40,17 @@ def post(self, request):
         if form.is_valid():
             feedback = Feedback(message=form.cleaned_data['message'])
             feedback.save()
-            return render_to_response("comparisontool/feedback_thanks.html",
-                                      locals(),
-                                      context_instance=RequestContext(request))
+            return render(request, "comparisontool/feedback_thanks.html")
```

Comment:
A style comment: while both single and double quotes are valid in Python, [PEP8](https://www.python.org/dev/peps/pep-0008/#string-quotes) suggests being consistent, so I would use single quotes here to match [below](https://github.com/cfpb/django-college-costs-comparison/pull/9/files#diff-638ddf7b5ed2ba01aee8e6b0f9d826d0R52). That being said, to quote from [Django's style guide](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/):

> Don’t waste time doing unrelated refactoring of existing code to conform to this style.